app/services/decisions/contact_decision_tree.rb

```diff
@@ -0,0 +1,12 @@
+module Decisions
+  class ContactDecisionTree < BaseDecisionTree
+    def destination
+      case step_name
+      when :home_details
```

Comment:
wondering if "home_address_details" would be less ambiguous? 🤔

Reply:
Yep, I changed it to `home_address` as it also is the name of the attribute where we are storing the address details, in the `contact_details` table.
`home_details` was a leftover from a previous incarnation of the code 🤣